CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Introduce hidden flag `--show-inputs-only` to show the inputs of the command
+that would be used if a command were executed.
```

Comment:
Based on discussion at standup, I think we agreed that making the option unhidden would be preferred, as long as there's an easy way to define it at the top level so that it applies to all commands but not command groups.

@gejohnston @awharn @zFernand0 Do you recall the same outcome of our discussion?

Reply:
Yes Timothy, the visibility of the flag as you described it was what I recall that Mike B advocated. In fact he said he would add that comment to this PR, but I suspect that he got sidetracked.